services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/extensions/dynamodb/mappingclient/functionaltests/BasicCrudTest.java

```diff
@@ -169,33 +171,33 @@ public int hashCode() {
     }
 
     private static final TableSchema<Record> TABLE_SCHEMA =
-        TableSchema.builder()
-                   .newItemSupplier(Record::new)
-                   .attributes(
-                       string("id", Record::getId, Record::setId).as(primaryPartitionKey()),
-                       string("sort", Record::getSort, Record::setSort).as(primarySortKey()),
-                       // This is a DynamoDb reserved word, forces testing of AttributeNames
-                       string("attribute", Record::getAttribute, Record::setAttribute),
-                       // Using tricky characters to force scrubbing of attributeName tokens
-                       string("*attribute2*", Record::getAttribute2, Record::setAttribute2)
-                           .as(secondaryPartitionKey("gsi_1")),
-                       string("attribute3", Record::getAttribute3, Record::setAttribute3)
-                           .as(secondarySortKey("gsi_1")))
-                   .build();
+        StaticTableSchema.builder()
+                         .newItemSupplier(Record::new)
+                         .attributes(
+                             fromString("id", Record::getId, Record::setId).as(primaryPartitionKey()),
```

Comment:
I'd like to have some discussion about whether this is semantically correct and what standard we should use here. Not necessarily disagreeing, just keeping this comment as a placeholder to make sure that discussion happens offline.